client/src/actions/authentication.js

```diff
@@ -0,0 +1,112 @@
+import { SIGNUP_ERROR, SIGNUP_PENDING, SIGNUP_SUCCESS, LOGIN_ERROR, LOGIN_PENDING, LOGIN_SUCCESS, CREATE_SESSION, CLEAR_SESSION } from '../constants';
```

Comment:
Minor: Might be more readable if you split this into a couple of different lines.